daemons/controld/controld_control.c

```diff
@@ -792,6 +798,22 @@ config_query_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, void
     crm_time_free(now);
 }
 
+/*!
+ * \internal
+ * \brief Trigger read and processing of the configuration
+ *
+ * \param[in] fn    Calling function name
+ * \param[in] line  Line number where call occurred
+ */
+void
+controld_trigger_config_processing(const char *fn, int line)
```

Comment:
This approach is used for `transition_trigger`, and it's convenient to use it for `config_read_trigger` and `fsa_trigger` too.